code-security/admin_guide/scan-monitor/secrets-scanning.adoc

```diff
@@ -0,0 +1,52 @@
+== Secrets Scanning
+
+You can use Code Security to detect and remove secrets in IaC files stored in your IDEs, Git-based VCS, and CI/CD pipelines.
```

Comment:
Can we add one screenshot of detecting a secret in an integration, such as the VS Code integration?